### ISSUE TITLE
Red map cant save

### ISSUE THREAD

**Issue by lynrayy** [mircokroon/minecraft-world-downloader#242]
**Describe the bug**
Sometimes world downloader cant resolve red zones and wont save it.
https://i.imgur.com/BiAQnkM.png

**Steps to reproduce**
Happens randomly

**Screenshots**
(https://i.imgur.com/BiAQnkM.png)

**Version**
 - Game: Minecraft 1.17.1
 - Downloader: latest
 - Launcher (if login issues): 14.2
 - Server play.spworlds.ru, SPZ gamemode

**Comment by mircokroon** [comment]
I think this might be entities moving around causing the chunk to get marked as unsaved again, is the chunk itself actually saved?

**Comment by lynrayy** [comment]
No, the time was wasted and the map (red zone) was not saved. But I found a partial way to fix such red zones - a rejoin to the server or teleportation to a hub / minigame, if this is such a server.
